### PR TITLE
[ADD]: to-do documentation

### PR DESCRIPTION
**Pull request by Donapi**
taskid-3193463

### CODE REVIEW

**Comment by Donapi** [comment]
@chiaraprattico I took your comments from PR5555 into account and created this new PR
@xpl-odoo could you please review? :)

File: content/applications/productivity/to_do.rst [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+Editing to-dos
+--------------
+
+Different actions are possible in a to-do:
+
+- :guilabel:`Tags`: select tags from the dropdown list to categorize your to-do. You can create a
+  new tag simply by writing it;
+- :guilabel:`Assignees`: select one or more users from the dropdown list to assign your to-do;
+- **Customize your tasks content**: when typing `/` in the text area, a dropdown list of features
+  related to the structure, format, media, navigation, widgets, banner, or basic blocks appear;
```
Comment (aju-odoo) [inline content/applications/productivity/to_do.rst:32]:
```suggestion
  related to the structure, format, media, navigation, widgets, banner, or basic blocks appears;
```

-> as it's the **list** of features that appear**s**

File: content/applications/productivity/to_do.rst [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+.. _to-do/convert-task:
+
+Converting to-dos into project tasks
+====================================
+
+You can convert a to-do into a task and assign it to a project by going to the to-do you want to
+convert, clicking the gear button (:guilabel:`⚙`) and :guilabel:`Convert to Task`. Select the
+:guilabel:`Project`, add or remove :guilabel:`Assignees`, select :guilabel:`Tags` from the project,
+and click :guilabel:`Convert to Task`. The to-do is now a project task and appears in the Project
+app under :guilabel:`My Tasks`.
```
Comment (aju-odoo) [inline content/applications/productivity/to_do.rst:97]: Once the to-do has been converted into a task, it appears in all of the menus of project, not only under 'my tasks'.

**Comment by xpl-odoo** [comment]
@robodoo r+

**Pull request closed without merging**